teachers_digital_platform/jinja2/teachers_digital_platform/crt-start.html

```diff
@@ -158,11 +158,16 @@ <h2>Curriculum information</h2>
                 <div class="m-notification
                             m-notification__visible
                             m-notification__warning">
-                    <span class="a-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1200" class="cf-icon-svg"><path d="M500 105.2c-276.1 0-500 223.9-500 500s223.9 500 500 500 500-223.9 500-500-223.9-500-500-500zm-49.7 234.6c0-27.6 22.4-50 50-50s50 22.4 50 50v328.6c0 27.6-22.4 50-50 50s-50-22.4-50-50V339.8zm50 582.5c-39.6 0-71.7-32.1-71.7-71.7s32.1-71.7 71.7-71.7S572 811 572 850.6s-32.1 71.7-71.7 71.7z"/></svg></span>
+                    <span class="a-icon"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1200" class="cf-icon-svg"><path d="M500 105.2c-276.1 0-500 223.9-500 500s223.9 500 500 500 500-223.9 500-500-223.9-500-500-500zm-49.7 234.6c0-27.6 22.4-50 50-50s50 22.4 50 50v328.6c0 27.6-22.4 50-50 50s-50-22.4-50-50V339.8zm50 582.5c-39.6 0-71.7-32.1-71.7-71.7s32.1-71.7 71.7-71.7S572 811 572 850.6s-32.1 71.7-71.7 71.7z"></path></svg></span>
                     <div class="m-notification_content">
                         <div class="m-notification_message">
                             <h3 class="h4">Limitations around saving your work</h3>
-                            <p>After you finish each dimension, we recommend you print the summary or save it as a PDF. Learn more about <button class="a-btn a-btn__link" onclick="openSaveWorkModalWindow();return false;">how to save your work.</a></p>
+                            <p>
+                                After you finish each dimension, we recommend you print the summary or save it as a PDF. Learn more about 
+                                <button class="a-btn a-btn__link" onclick="openSaveWorkModalWindow();return false;">how to save your work.
```

Comment:
Should `return false` use `event.preventDefault instead? https://stackoverflow.com/questions/128923/whats-the-effect-of-adding-return-false-to-a-click-event-listener?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa